lapis/storage.py

```diff
@@ -0,0 +1,199 @@
+from typing import Optional, NamedTuple
+
+from usim import time, Resources, Pipe, Scope
+
+from lapis.files import StoredFile, RequestedFile
+
+
+class LookUpInformation(NamedTuple):
+    cached_filesize: int
+    storage: "Storage"
+
+
+class RemoteStorage(object):
+    def __init__(self, pipe: Pipe):
+        self._connection = pipe
+
+    async def transfer(self, total, job_repr):
+        await self._connection.transfer(total=total)
```

Comment:
@tfesenbecker: Please revert this change you introduced in the last commit. The intention is to have the **same** API for all Storage objects!

Reply:
Yes, but please be aware that in this case we will have different function calls in https://github.com/MatterMiners/lapis/blob/27c1ea7c81d688625edc23a015570bb84f1d9104/lapis/storage.py#L183-L188.

Reply:
I see your point but in order to do this I would have to mess up the RequestedFile's attributes in order to use the RemoteStorage connection like that:
https://github.com/MatterMiners/lapis/blob/27c1ea7c81d688625edc23a015570bb84f1d9104/lapis/storage.py#L183-L188

I'm not sure what is worse in this case. Let's talk about this later